README.md

```diff
@@ -29,7 +29,7 @@ with tfe.protocol.Pond(server0, server1, crypto_producer) as prot:
     inputs = [prot.define_private_input(inputter) for inputter in inputters]
 
     # securely sum all inputs and divide by count
-    result = reduce(lambda x,y: x+y, inputs) / len(inputs)
+    result = reduce(lambda x,y: x+y, inputs) * (1 / len(inputs))
```

Comment:
want to implement (public) division as a fix instead? :D it shouldn't be that hard: you could simply take the inverse when the divisor is passed to `lift`

Reply:
happy to help of course